tests/Event/EventCommandHandlerTest.php

```diff
@@ -141,7 +141,7 @@ public function it_can_update_major_info_of_an_event(): void
             ->when(
                 new UpdateMajorInfo($id, $title, $eventType, $location, $calendar)
             )
-            ->then([new MajorInfoUpdated($id, $title, $eventType, $location, $calendar)]);
+            ->then([new MajorInfoUpdated($id, $title->toString(), $eventType, $location, $calendar)]);
```

Comment:
I think the constructor can be changed for `$title` to a primitive string instead of the `Title` class